packages/react-router-dom/modules/__tests__/BrowserPrompt-test.js

```diff
@@ -0,0 +1,32 @@
+import expect from 'expect'
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { MemoryRouter } from '../../'
+import { Router } from '../../'
+import { Route } from '../../'
+import { BrowserPrompt } from '../../'
```

Comment:
These are importing from the build files, which I believe would require an `npm build` to be run before the tests are run. They should be importing from the source files, i.e., `import { MemoryRouter } from '../MemoryRouter'`